lightning/src/util/events.rs

```diff
@@ -203,6 +203,8 @@ pub enum Event {
 		all_paths_failed: bool,
 		/// The payment path that failed.
 		path: Vec<RouteHop>,
+		/// The channel responsible for the failed payment path.
+		short_channel_id: Option<u64>,
```

Comment:
Forgot to mention -- per Matt's comment [here](https://github.com/rust-bitcoin/rust-lightning/pull/1077#discussion_r726557152) and any other cases, could we explain when this field should be set? I had previously assumed it was only a Option for backwards compat

Reply:
Also, I think a more descriptive field name might be `blameable_scid`?

Reply:
I'd prefer to leave the naming as is presuming more descriptive docs, as requested. In the context of `PaymentPathFailed` there's not many ways this can be interpreted.